lib/tasks/data/supplier_locations.yml

```diff
@@ -243,51 +413,99 @@ suppliers:
     - DC5
     - DC6
     - DC8
+    - DC9
     - DST1
+    - DST10
+    - DST11
+    - DST12
+    - DST13
     - DST2
     - DST3
+    - DST5
+    - DST7
+    - DST8
+    - DST9
     - DWI
     - EEI
     - EHI
     - ESI
     - ESX1
+    - ESX10
+    - ESX11
+    - ESX12
+    - ESX13
+    - ESX14
+    - ESX15
+    - ESX16
+    - ESX17
+    - ESX18
     - ESX2
     - ESX3
     - ESX4
     - ESX5
     - ESX6
     - ESX7
+    - ESX8
+    - ESX9
     - EWI
     - EXI
     - EYI
     - FDI
     - FMI
     - FYI
     - GCS1
+    - GCS2
+    - GCS3
+    - GCS4
     - GCS5
+    - GCS6
     - GMI
     - GNI
     - HBI
     - HCI
+    -

```

Comment:
Not sure if having this blank line will cause an issue? Probably best to delete just to be safe.

Reply:
whoops, that should not be there, good spot